.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### Expected behavior vs actual behavior
+<br><br>
```

Comment:
Line spacing to make it easy to move the cursor directly to where you want to type. I couldn't find a way to get it to do that in GitHub's markdown. If there's a way I'm all for changing it.